### PR TITLE
feat: Add table and label component

### PR DESCRIPTION
**Pull request by duanyytop**
The components impact many pages, so there are many file changes in this pr.

But there are a lot places which need to be refactored, I just submit firstly to avoid too long code.

### CODE REVIEW

File: web/src/pages/Address/index.tsx [hunk 1]
```diff
@@ -188,54 +84,32 @@ export default (
...
-              label="Transactions: "
-              value={`${AddressData.data.transactions_count}`}
-              style={{
-                position: 'relative',
-                top: -28,
-                left: 583,
-              }}
-            />
-            <AddressCellConsumedLabel
+            <AddressCommonRowPenal>
```
Comment (wuyuyue) [inline web/src/pages/Address/index.tsx]: panel

File: web/src/pages/Address/index.css.tsx [hunk 2]
```diff
@@ -186,3 +166,10 @@ export const AddressTransactionsCell = styled.div`
 export const AddressTransactionsPagition = styled.div`
   margin: 80px 0 200px 0;
 `
+
+export const AddressCommonRowPenal = styled.div`
```
Comment (wuyuyue) [inline web/src/pages/Address/index.css.tsx]: panel

File: web/src/pages/Address/index.tsx (hunk 1 above)
Comment (duanyytop) [reply web/src/pages/Address/index.tsx]: OK

**Comment by duanyytop** [comment]
Solved. Please review again. Thanks

**Pull request merged**